demos/mcp_server_memory/db/sqlite_manager.py

```
@@ -0,0 +1,581 @@
+            return False
+
+    def list_summary_types_by_memory_id(self, memory_id: str, ) -> List[Dict[str, Any]]:
+        """List all summaries of this memory in the database.
+
+        Returns:
+            List[Dict[str, Any]]: List of summary types and their counts
+        """
+        try:
+            with SQLiteConnection(SQLITE_PATH) as conn:
+                cursor = conn.cursor()
+                summaries = cursor.execute(
```

Ruff (0.11.9)

Local variable `summaries` is assigned to but never used

Remove assignment to unused variable `summaries`

(F841)